web/src/narrow.js

```diff
@@ -372,8 +372,8 @@ export function activate(raw_terms, opts) {
             id_info.target_id = opts.then_select_id;
             // Position selected row to not scroll off-screen.
             if (opts.then_select_offset === undefined) {
-                const $row = message_lists.current.get_row(opts.then_select_id);
-                if ($row.length > 0) {
+                const $row = message_lists.current?.get_row(opts.then_select_id);
+                if (message_lists.current !== undefined && $row.length > 0) {
```

Comment:
I think you could do this as `if (opts.then_select_offset === undefined && message_lists.current !== undefined)` above...

Reply:
Yeah, changed.